### DISCUSSION TITLE
slack bot integration with docusaurus

### DISCUSSION THREAD

**Discussion by vivekVells** [facebook/docusaurus#8157] [Q&A]
Hi all, trying to figure out if there is a plugin or any docs link on how to integrate slack bot with docusaurus. 

unable to find a good reference on this. any thoughts?

updated: trying to figure out for a use case: if new blogs got published in out docusaurus site, a notification is automatically posted in our slack channel via a slack bot.

**Comment by slorber** [comment]
We don't provide such resources, and even have no idea what a slack bot for Docusaruus would be expected to do. 

It's impossible to help you because you don't describe much what you try to achieve. You could find help on our Discord server

**Reply by vivekVells** [reply]
sorry, @slorber. should have provided more info. 

updated the discussion description 

Trying to figure out for a use case: if new blog posts got published in our docusaurus site, a notification is automatically posted in our slack channel via a slack bot. 

Please let me know if you have any thoughts on this. 

(will also search thro' discord)

**Reply by slorber** [reply]
Docusaurus blogs have an RSS feed by default, and you'll likely find a lot of tooling to post RSS feed items to Slack

https://slack.com/help/articles/218688467-Add-RSS-feeds-to-Slack
